.github/workflows/release-drafter.yml

```diff
@@ -1,41 +1,57 @@
 name: Release Drafter
 
 on:
-  push:
-    # branches to consider in the event; optional, defaults to all
-    branches:
-      - master
-  # pull_request event is required only for autolabeler
-  pull_request:
-    # Only following types are handled by the action, but one can default to all as well
-    types: [opened, reopened, synchronize]
-  # pull_request_target event is required for autolabeler to support PRs from forks
-  # pull_request_target:
-  #   types: [opened, reopened, synchronize]
+  workflow_call:
```

Comment:
ok to have this pipeline invokable for other repos; but now it seems as if we have no trigger for the releases of these 'github-workflows' itself. 
I'd add another pipeline for this ... just using this template as everyone else does.

Reply:
Hi @ivy-rew , I just created a new pipeline self-release-drafter to create the release note for github-workflows project. Please help me to re-check it. Thanks!